films/models.py

```diff
@@ -0,0 +1,99 @@
+from platform import release
+from django.db import models
+
+# 영화 테이블
+
+class Film(models.Model):
+    name             = models.CharField(max_length=100)
+    release_date     = models.DateField()
+    image_url        = models.URLField()
+    running_time_min = models.CharField(max_length=20, default='')
+    descriptions     = models.CharField(max_length=1000, default='')
+    rating_systems   = models.ForeignKey('RatingSystem', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'films'
+
+# 일대다 관계 테이블
+
+class RatingSystem(models.Model):
+    rate = models.CharField(max_length=50, default='')
+
+    class Meta:
+        db_table = 'rating_systems'
+
+# 다대다 관계 테이블
+
+class Genre(models.Model):
+    name  = models.CharField(max_length=50)
+    films = models.ManyToManyField(Film, through='FilmGenre')
```

Comment:
M2M field에 related_name을 추가해 주시면 좋을 것 같습니다.
```suggestion
    films = models.ManyToManyField(Film, through='FilmGenre', related_name='genres')
```